### ISSUE TITLE
Blurry on Windows 10

### ISSUE THREAD

**Issue by dumbcoder7** [trelby/trelby#4]
I downloaded Trelby from the official website. However, the installed software appears quite blurry in my Windows 10 computer. I had to disable the Windows of 'Scale apps automatically' to reduce the blur, but it still exists.
Will there be any solution to this in the near future?

Adding a screenshot.

![image](https://user-images.githubusercontent.com/43912470/100938029-f2ddaa80-3519-11eb-972b-f8e3e2fed6d6.png)

I hope you will agree that it can be a bit smoother than now.

**Comment by twinysam** [comment]
Last commit was on October 2016. I think it's fair to assume that Trelby is dormant.

**Comment by limburgher** [comment]
FYI I have a PR open for a port to Python 3, and I'm maintaining it at https://github.com/limburgher/trelby

**Comment by dumbcoder7** [comment]
The README doesn't tell me much. Consider updating that and enabling the `Issues` section.

**Comment by limburgher** [comment]
Done, and done. I don't yet have a Windows build ready but I hope to soon.

**Comment by JimmyCushnie** [comment]
I can confirm this issue. Trelby is blurry for me on Windows 10 with a 4k screen.

![image](https://user-images.githubusercontent.com/32693298/120974783-89e02500-c73e-11eb-87c1-52ed706275ff.png)

**Comment by adityaraute** [comment]
Yes but sadly this repo is pretty inactive. I am waiting for @limburgher 's Windows build. Until then I am using Scenarist, which is actually pretty cool, too.

**Comment by JimmyCushnie** [comment]
Ooh, Scenarist looks cool, thanks for the suggestion. Personally I've settled on Scrite and I'm very happy.

**Comment by anilgulecha** [comment]
Wow, scrite looks fantastic. Going to try it out today.

I recall I first started working on Trelby since there was no good viable
alternative. Life took over since then and have not been able to put more
time since v2.2.

But with software like Scrite now available, FOSS, and more multi-platform,
I think we can say that we have solid options around.

**Comment by AvirFrog** [comment]
@anilgulecha 

Why don't you adapt to the changes made by @limburgher? Rewriting Trelby to Python  3.X allows you to find more people to develop your program. Who knows how many versions of Trelby are currently better than the official version?

**Comment by xucaen** [comment]
The screenshots that are supposed to be blurry are not blurry for me. Trelby is also not blurry on my windows 10 and windows 11 installations.

Blurry apps in windows 10 is a windows issue. I found this link to resolve the issue: https://support.microsoft.com/en-us/windows/fix-apps-that-appear-blurry-in-windows-10-e9fe34ab-e7e7-bc6f-6695-cb169b51de0f

**Comment by NotASKamDev** [comment]
Still blurry, and the link isn't changing anything
